nebu/tests/cli/test_get.py

```diff
@@ -193,7 +193,7 @@ def test_failed_request_omitting_version(self, invoker):
 
         assert result.exit_code == 2
 
-        assert 'Missing argument "col_version"' in result.output
+        assert 'missing argument "col_version"' in result.output.lower()
```

Comment:
I had to convert the expectation to lowercase to prevent an error on Mac OS where `"col_version"` in `Missing argument error` is returned as `"COL_VERSION"`.